packer/apply_cis_rules

```diff
@@ -247,3 +247,28 @@ MACs hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512,h
         file.write("readonly TMOUT\n")
         file.write("export TMOUT\n")
     os.chmod('/etc/profile.d/tmout.sh', 0o755)
+
+    # xccdf_org.ssgproject.content_rule_accounts_umask_etc_bashrc
+    bash_bashrc = '''
+
+umask 027
+    '''[1:-1]
+    with open('/etc/bash.bashrc', 'a') as f:
+        f.write(bash_bashrc)
+
+    # xccdf_org.ssgproject.content_rule_accounts_umask_etc_login_defs
+    with open('/etc/login.defs') as f:
+        login_defs = f.read()
+    login_defs = re.sub(r'^UMASK\s+022', 'UMASK           027', login_defs, flags=re.MULTILINE)
+    with open('/etc/login.defs', 'w') as f:
+        f.write(login_defs)
+
+    # xccdf_org.ssgproject.content_rule_accounts_umask_etc_profile
+    umask_sh = '''
+#!/bin/bash
+umask 027
+    '''[1:-1]
+    with open('/etc/profile.d/99-umask.sh', 'w') as f:
+        f.write(umask_sh)
+    os.chmod('/etc/profile.d/99-umask.sh', 0o755)
+>>>>>>> cf9528f (Ensure that Users Have Sensible Umask Values)
```

Comment:
this looks like a leftover from a merge conflict

Reply:
fixed